interfaces/builtin/docker_support.go

```diff
@@ -609,7 +625,7 @@ func (iface *dockerSupportInterface) AppArmorConnectedPlug(spec *apparmor.Specif
 	if !release.OnClassic {
 		spec.AddSnippet(dockerSupportConnectedPlugAppArmorCore)
 	}
-	spec.UsesPtraceTrace()
+	spec.SetUsesPtraceTrace()
```

Comment:
Ouch, that's a bug!

Reply:
Yep, here and below. I recall that there was a change in the API for the Getter and the Setter during the review. Guess we missed it :\